### ISSUE TITLE
Text element crashes if no fontFamily specified

### ISSUE THREAD

**Issue by glennsl** [revery-ui/revery#622]
On Xorg/Linux, if no `fontFamily` is specified on a `Text` element it will creash with the following error:
```
Fatal error: exception Invalid_argument("\"\": invalid path")
Raised at file "stdlib.ml", line 30, characters 20-45
Called from file "src/Path.re", line 20, characters 14-26
Called from file "src/Core/Environment.re", line 87, characters 6-30
Called from file "src/Draw/FontCache.re", line 61, characters 18-52
Called from file "src/Draw/Text.re", line 42, characters 13-55
Called from file "src/UI/TextNode.re", line 127, characters 6-65
```

This happens because the default `fontFamily` on `Style.t` is `""`, which apparently is considered an invalid path somewhere. Replacing the default with any random bunch of alpha-numeric character avoids the crash, but of course is unlikely to return a valid font and therefore doesn't render anything.

I'd consider this a bug in `reason-font-manager` first and foremost, but unless `""` has some special significance with `reason-font-manager` and is supposed to load the default font, the behavior in revery doesn't seem great either. It ought to load some kind of reasonable default.

**Comment by SirFurness** [comment]
It might be worth looking at what was mentioned in #143 

Also, in case it helps anyone, I went looking and the invalid path error is coming from
https://github.com/revery-ui/rench/blob/a976fe51b89b4943e8650f8b9d6ebc49331678b0/src/Path.re#L20

In Fpath, `Fpath.v` calls an `of_string` function which the docs [here](https://erratique.ch/software/fpath/doc/Fpath.html#VALof_string) say returns an error if the string is empty.

**Comment by SirFurness** [comment]
The update to font families #879 fixes this issue, so this can be closed along with #143.
